ssl/ssl_asn1.c

```diff
@@ -194,7 +194,7 @@ int i2d_SSL_SESSION(SSL_SESSION *in, unsigned char **pp)
     }
     if (in->ext.tick_lifetime_hint > 0)
         as.tlsext_tick_lifetime_hint = in->ext.tick_lifetime_hint;
-    as.tlsext_tick_age_add = in->ext.tick_age_add;
+    as.tlsext_tick_age_add = (long)(int32_t)in->ext.tick_age_add;
```

Comment:
Perhaps a silly question, but I imagine that `int32_t` gets automatically converted to `long` with no need for the extra help?